src/test/java/DeadlineTest.java

```diff
@@ -0,0 +1,11 @@
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class DeadlineTest {
+    @Test
+    public void testOne() {
```

Comment:
good Test. You can include more test in this, for instance if you mark it as isDone will the boolean attribute change!